src/Bicep.Core/Emit/EmitLimitationCalculator.cs

```diff
@@ -729,60 +732,10 @@ private static ImmutableDictionary<ExtensionConfigAssignmentSymbol, ImmutableDic
                 }
             }
 
-            var evaluator = new ParameterAssignmentEvaluator(model);
-            HashSet<Symbol> erroredSymbols = new();
+            var extensionConfigAssignmentSymbols = model.Binder.Bindings.Values.OfType<ExtensionConfigAssignmentSymbol>();
 
-            foreach (var symbol in GetTopologicallySortedSymbols(referencesInValues))
+            foreach (var extConfigAssignment in extensionConfigAssignmentSymbols)
             {
-                if (symbol.Type is ErrorType)
```

Comment:
I removed most of this. In the invalid baselines there are several tests for extension config <-> param,var references which will currently not work.